wpilib/wpilib/_impl/main.py

```diff
@@ -9,6 +9,12 @@
 from .logconfig import configure_logging
 import hal_impl
 
+from wpilib.driverstation import DriverStation
+
+def my_except_hook(exctype, value, traceback):
+    DriverStation.reportError("ERROR Unhandled {}".format(exctype), True)
+    sys.__excepthook__(exctype, value, traceback)
+sys.excepthook = my_except_hook     
```

Comment:
Hm, seems like you would want to do this only when the code is actually launched, instead of always.

Reply:
you mean if 'run' is the entry point or whatever?

Reply:
Well. Maybe only when the user's code is about to be launched?